autokeras/hypermodel/hyper_block.py

```diff
@@ -69,34 +70,22 @@ class RNNBlock(HyperBlock):
 
     def build(self, hp, inputs=None):
         input_node = layer_utils.format_inputs(inputs, self.name, num=1)[0]
+        feature_size = input_node.shape[2]
```

Comment:
add a check first, if len(input_node.shape) < 3: raise ValueError('The input to RNNBlock must have at least 3 dimensions').

Reply:
Should we check for < 3 or strictly == 3. To enforce 3-D shape i believe we need the second